tests/conftest.py

```diff
@@ -2242,3 +2250,38 @@ def mock_config(rnafusion_dir: Path, rnafusion_case_id: str) -> None:
     Path(rnafusion_dir, rnafusion_case_id, f"{rnafusion_case_id}_samplesheet.csv").touch(
         exist_ok=True
     )
+
+
+@pytest.fixture
+def store_with_sequencing_metrics(store: Store):
+    """Return a store with multiple samples with sample lane sequencing metrics."""
+
+    sample_sequencing_metrics_details = [
+        ("sample1", "flowcell1", 1, 1000000, 90.5, 32),
```

Comment:
```suggestion
        ("sample_1", "flow_cell_1", 1, 1_000_000, 90.5, 32),
```
I think this is easier to read